src/combineReducers.js

```diff
@@ -74,6 +74,48 @@ function assertReducerSanity(reducers) {
   })
 }
 
+function getReducers(config) {
+  var reducerKeys = Object.keys(config)
+  var reducers = {}
+
+  for (var i = 0; i < reducerKeys.length; i++) {
```

Comment:
While it is true in PHP, **it is not for JS**. Modern JS VMs optimize this case and are faster without this trick. @mraleph wrote about it: [How the Grinch stole array.length access](http://mrale.ph/blog/2014/12/24/array-length-caching.html).